src/main/java/com/redhat/pantheon/sling/PantheonRepositoryInitializer.java

```diff
@@ -41,22 +41,29 @@ public void processRepository(SlingRepository slingRepository) throws Exception
     private void initializeRepositoryACLs(SlingRepository slingRepository) throws RepositoryException {
         JackrabbitSession s = (JackrabbitSession) slingRepository.loginAdministrative(null);
         try {
-            User admin = (User) s.getUserManager().getAuthorizable("admin");
-            admin.changePassword("ccsadmin"); // FIXME - hardcoding admin passwords is a Bad Thing
```

Comment:
Did you remove this because you reimplemented it elsewhere, or did you just want it gone?

Reply:
I just wanted it gone :blush:  (I just think we shouldn't be hardcoding passwords in the code)

Reply:
...as you also realized in your FIXME comment.

Reply:
Yes, hardcoding it was Bad, but it was Less Bad than leaving it as the default!

Well, it's a small issue, especially if we're sending the users through our custom login form that doesn't suggest admin:admin right next to the login fields...

I'm sure there is a "proper" solution that we'll figure out someday, but perhaps we should record that as a jira issue so we don't forget about it.